react/components/ColorPicker/Gradient.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { CustomPicker } from 'react-color'
+import { Hue } from 'react-color/lib/components/common'
+
+/**
+ * Gradient Component
+ */
+class Gradient extends React.Component {
+  /**
+   * Render Gradient Component
+   */
+  render() {
+    return (
+      <div className="gradient-container relative w-100 mv6">
+        <Hue
+          {...this.props}
+          onChange={this.props.onChange}
+          pointer={() => (
+            <div
+              className="pointer-gradient"
+              style={{ background: this.props.hex }}
+            />
+          )}
+          direction="horizontal"
+        />
+      </div>
+    )
+  }
+}
+
+Gradient.propTypes = {
+  /** onChange event */
+  onChange: PropTypes.func,
+  /** Pointer Hex Color */
+  hex: PropTypes.object,
```

Comment:
É um objeto mesmo? Ta sendo usado dentro de `style={{ background: this.props.hex }}`

Reply:
Opa não vi isso, na verdade seria uma `string`.